backend/backend/charge_rent.py

```diff
@@ -0,0 +1,44 @@
+"""Module enables users to be charged rent
+   when they land on owned properties
+"""
+
+import json
+import sys
+import cgitb
+from backend.player import Player
+from backend.game import get_games
+from backend.properties import Property
+
+cgitb.enable()
+
+
+def charge_rent(source=sys.stdin, output=sys.stdout):
+    """Entry point for a player to be charged
+       rent and property owner gains rent amount
+    """
+
+    output.write('Content-Type: application/json\n\n')
+    request = json.load(source)
+    player_id = request["player_id"]
+    games = get_games()
+
+    # Access the game the player is playing in
+    # and the position the player is on the board
+    with Player(player_id) as player:
+        position = player.board_position
+        for game in games:
+            if player.username in games[game]:
+                game_id = game
```

Comment:
I think `game_id` should be initialised outside this for-loop with None because variables initialised inside a loop only live for the scope of the loop. You could put `game_id = None` under `games = get_games()`.